docs/cheaha/open_ondemand/index.md

```diff
@@ -25,3 +25,40 @@ For a more complete description of how to select resources, go [here](ood_layout
 ## Debugging OOD Job Failures
 
 If your OOD job cards are disappearing after being allocated or during the job, see our [documentation](ood_layout.md#debugging-ood-job-failures) for instructions on how to retrieve the logs and submit a ticket to Research Computing support.
+
+## Choosing the right Jupyter Application
+
+Jupyter environments have revolutionized the way researchers, analysts, and developers interact with code. Whether you have workflows in Python, visualizing genomic data, running simulations or run any kind of computational workflow on our high-performance computing (HPC) cluster, you've likely encountered using [Jupyter Notebook](./ood_jupyter.md) and/or [JupyterLab](./ood_jupyterlab.md), we have detailed information for using both on Cheaha. Both applications are open-source web tools designed for creating and sharing documents that can include your code, equations, visualizations, and narrative text. Their main purpose is to offer an interactive environment for all phases of your data analysis life cycle including, data exploration, analysis, and visualization. But when should you choose one over the other on Cheaha to run your computational workflow?
+
+This article tries to help you decide, by comparing both Jupyter Apps, looking at their strengths and weaknesses, and how they measure up to your needs, especially on Cheaha.
+
+### What They Have in Common
+
+Before listing out some of the differences, the following are similarities they have in common;
+
+1. They use the same underlying kernel system. (i.e. environments created for use with Jupyter Notebook can also be used in JupyterLab and vice-versa).
+1. They both support `.ipynb` notebooks.
+1. Allow interactive code execution, plotting, markdown, and data visualization.
+1. Can run Python, R, Julia, and other supported languages.
```

Comment:
Be consistent with the "They". Or pull that part out to the start of the list like

"They both:"

Then in each list item, don't start with an initial capital. Either way you choose is fine, be consistent.